### DISCUSSION TITLE
Raycast Extension for Fabric

### DISCUSSION THREAD

**Discussion by AlexMC** [danielmiessler/Fabric#1104] [Show and tell]
Hi everyone, 

I created a small Raycast extension for personal use with a lot of help from Cursor and Sonet. 
Feel free to use it as you wish! :)  
https://github.com/AlexMC/fabric-raycast-extension 

![Demo](https://raw.githubusercontent.com/AlexMC/fabric-raycast-extension/master/.github/assets/demo.gif)

**Comment by noamsiegel** [comment]
This is awesome! 

I can't seem to find it in the raycast extension store — when will it be available? Or does it only work through `npm install`?

**Reply by AlexMC** [reply]
Yep, only development mode for now. Raycast extensions are strongly curated, probably hard to get this into the store.

**Reply by treyg** [reply]
> Yep, only development mode for now. Raycast extensions are strongly curated, probably hard to get this into the store.

I bet you'd have no problem getting it in the store. You should try!

**Reply by larsommen** [reply]
If you give it a try you might get some insight to why not - if not.

**Reply by treyg** [reply]
For a fabric extension specifically or just any extension?

**Reply by larsommen** [reply]
I was thinking of the fabric extension but I guess it goes for any 🤷🏻‍♂️😊

**Reply by treyg** [reply]
@larsommen Gotcha! I was just curious if there was a specific reason they wouldn't allow a fabric extension. I have published a different extension though, so I know what you mean. It can be a quite a process.

**Comment by gchingon** [comment]
does this use Raycast ai or fabric --setup ai choice?
